package.json

```diff
@@ -23,6 +23,7 @@
     "prepare": "gulp",
     "preversion": "npm test",
     "release": "gulp release",
+    "federalist": "npm install && npx patch-package && npx gulp",
```

Comment:
consider adding the patch as a `postinstall` https://docs.npmjs.com/cli/v6/using-npm/scripts#npm-install
which would prevent us from having to include in the chain command here and as well for running after install manually when getting started locally

```suggestion
    "postinstall": "npx patch-package",
    "federalist": "npm install && npx gulp",
```

Reply:
Added it, thank you!